component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -331,6 +359,74 @@ public static long getTimeStepSize(AuthenticationContext context) {
 		}
 	}
 
+	/**
+	 * Get stored time step size.
+	 *
+	 * @param tenantDomain Tenant domain name.
+	 * @return Time step size.
+	 * @throws TOTPException On Error while getting value for time step size from registry.
+	 */
+	public static long getTimeStepSizeFromRegistry(String tenantDomain, AuthenticationContext context)
+			throws TOTPException {
```

Comment:
Not formatted

Reply:
addressed with https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/77/commits/9441e9b18f9e89e6d4184cf0beef797878c68e7d